tests/InputFiles/GeneralizedHarmonic/BinaryBlackHole.yaml

```diff
@@ -30,6 +30,7 @@ DomainCreator:
     EnvelopingCube:
       Radius: 100.0
       UseProjectiveMap: true
+      Sphericity: 1.0
```

Comment:
Please keep this at `0.0` for now, just to make sure people using this input file don't get confused that the domain changed.